tests/core/pthread/create.cpp

```diff
@@ -48,7 +48,7 @@ void mainn() {
 
 int main() {
   // Create initial threads.
-  for(int i = 0; i < NUM_THREADS; ++i) {
+  for(long i = 0; i < NUM_THREADS; ++i) {
```

Comment:
I think you can leave this as `int`, no?

Reply:
I was just trying to make the code work with as few casts as possible. Since the thread id is stored as a long/pointer, it seemed nicer to make this loop use `long` instead of `int`.